content/04.network_applications.md

```diff
@@ -39,5 +39,27 @@
 
 #### Clinical applications
 
-1. Can mention EHR use and other related applications
-2. Mention Tiffany's work on private data embeddings
+Using knowledge graphs for clinical applications is in early stages of development and these graphs are used to improve patent healthcare.
+Typically, these graphs are constructed from electronic health records (EHR) where nodes represent patients, drugs and diseases and edges represent relationships such as a patient being perscribed a treatment or a patient being diagnosed with a disease [@pmid:26306276; @doi:10.1145/2110363.2110415; @arxiv:1707.05340; @doi:10.1186/s12859-015-0549-5].
+Tasks in this context range from improving patient diagnoses [@doi:10.1109/TKDE.2016.2605687;@arxiv:1709.06908] to recommending safer drugs for patients [@arxiv:1710.05980; @arxiv:1709.06908].
+We discuss approaches that project knowledge graphs into a low dimensional space to make clinical predictions.
```

Comment:
```suggestion
```
I feel like this is in each of the other sections too, so I went ahead and took it out here. Maybe it would be better to move this sentiment to the overall section header since it applies to each.